qiita_pet/static/js/sharing.js

```diff
@@ -1,5 +1,3 @@
-var current_study = null;
-
 $(document).ready(function () {
```

Comment:
Because you can have as many document.ready calls in a single page as needed, and this also standardizes naming and initialization of the sharing dropdown for use in the other helper functions.